cadc-log/src/main/java/ca/nrc/cadc/log/LogControlServlet.java

```diff
@@ -560,9 +562,8 @@ void isAuthorizedGroup(Authorizer authorizer, Subject subject, boolean readOnly)
      *
      * @return Set of authorized groupURI's, can be an empty Set if none configured.
      */
-    Set<String> getAuthorizedGroupUris() {
+    Set<String> getAuthorizedGroupUris(PropertiesReader propertiesReader) {
```

Comment:
This should return Set of GroupURI

Reply:
Updated to use the GroupClient to validate group membership using the properties file, and return a list of GroupURI's when reading the groups in the properties file.